CodeEdit/Features/CodeEditUI/Views/KeyValueTable.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  KeyValueTable.swift
+//  CodeEdit
+//
+//  Created by Abe Malla on 5/16/24.
+//
+
+import SwiftUI
+
+struct KeyValueItem: Identifiable {
+    var id: String { key }
+    let key: String
+    let value: String
+}
+
+private struct NewListTableItemView: View {
+    @State private var key = ""
+    @State private var value = ""
+    var completion: (String, String) -> Void
+
+    var body: some View {
+        VStack {
+            Text("Enter new key-value pair")
```

Comment:
Could you provide better instructions on what to enter into the text fields, e.g., `path`?